docs/configuration/http_conn_man/route_config/rate_limits.rst

```diff
@@ -102,12 +110,88 @@ Remote Address
     "type": "remote_address"
   }
 
-The following descriptor is sent using the trusted address from :ref:`x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>`:
+The following descriptor entry is appended to the descriptor and is populated using the trusted
+address from :ref:`x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>`:
+
+    * ("remote_address", "<:ref:`trusted address from x-forwarded-for
+      <config_http_conn_man_headers_x-forwarded-for>`>")
+
+Rate Limit Key
+^^^^^^^^^^^^^^
+
+.. code-block:: json
+
+  {
+    "type": "rate_limit_key",
```

Comment:
I think I would just rename this "key" or "generic_key"

To me "rate_limit_ is also redundant here and somewhat verbose.